packages/@react-spectrum/button/stories/ActionButton.stories.tsx

```diff
@@ -96,3 +100,31 @@ function render(props = {}) {
     </Flex>
   );
 }
+
+// https://codesandbox.io/s/eloquent-herschel-pgwci?file=/index.html
+function DisabledButtonBug() {
+  let [fooDisabled, setFooDisabled] = useState<boolean>();
+  let fooPressStart = () => {
+    console.log('press start foo');
```

Comment:
nit: what do you think about replacing the console.logs here w/ `action('fooPressStart')()`? I find it hard to remember what stories fire console logs vs actions when testing

Reply:
that's fair, i can do it to the rest of them